AsyncDisplayKit/Details/ASDataController.mm

```diff
@@ -462,6 +462,23 @@ - (void)_reloadDataWithAnimationOptions:(ASDataControllerAnimationOptions)animat
   }];
 }
 
+- (void)waitUntilAllUpdatesAreCommitted
+{
+  ASDisplayNodeAssertMainThread();
```

Comment:
Since we assert main here, probably can remove the assert from the other two places, but not a big deal / could do that with later cleanup (every line counts in huge files with lots of asserts!)